deps/deps.go

```diff
@@ -7,6 +7,7 @@ import (
 	_ "github.com/gophercloud/gophercloud/openstack/blockstorage/extensions/quotasets"
 	_ "github.com/gophercloud/gophercloud/openstack/blockstorage/v3/volumes"
 	_ "github.com/gophercloud/gophercloud/openstack/compute/v2/extensions/quotasets"
+	_ "github.com/gophercloud/gophercloud/openstack/compute/v2/servers"
```

Comment:
This is not required as we use this already somewhere else. we only need to list stuff here is that is exclusively used in tests

Reply:
I removed this from PR.